persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml

```diff
@@ -1,2 +1,4 @@
 [persist_generate_100.entities.redis]
 connection = "redis://localhost:6379"
+
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/ballerina-platform/persist-tools/pull/341/commits/06fbbe450e1f29e029b4f077d2da6c4a1e1bb9d8